src/screens/MixpanelHook.res

```diff
@@ -34,9 +34,21 @@ let useSendEvent = () => {
 
   let mixpanel_token = Window.env.mixpanelToken
 
-  let trackApi = async (~email, ~merchantId, ~description, ~event) => {
+  let url = RescriptReactRouter.useUrl()
+  let endpoint = url.path->List.toArray->Array.get(1)->Option.getOr("")
```

Comment:
Does this ignore `/dashboard` as the parameter ? If product decided to remove /dashboard from the url then it would give false results . There's something called as dashboardBasePath in Global vars . Please check if value is present take the 1st value or else the 0th value . If theres a way which doesn't require to get the index then can we implement that

Reply:
okay, updated the PR, thanks.